code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -95,6 +99,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 				"disabilities" = pref.disabilities,
 				"organs_data" = list2params(pref.organ_data),
 				"organs_robotic"= list2params(pref.rlimb_data),
+				"body_markings" = json_decode(pref.body_markings),

```

Comment:
`"body_markings" = json_encode(pref.body_markings)`

encode to turn a list into text. decode to turn text into a list. We store text on the database, we use a list ingame.